pkgs/by-name/im/immer/package.nix

```diff
@@ -1,25 +1,33 @@
 {
   lib,
   stdenv,
+  catch2,
   fetchFromGitHub,
   cmake,
+  nix-update-script,
 }:
 
 stdenv.mkDerivation rec {
   pname = "immer";
-  version = "0.8.0";
+  version = "0.8.1";
 
   src = fetchFromGitHub {
     owner = "arximboldi";
     repo = "immer";
-    rev = "v${version}";
-    hash = "sha256-R0C6hN50eyFSv10L/Q0tRdnUrRvze+eRXPrlAQsddYY=";
+    tag = "v${version}";
+    hash = "sha256-Tyj2mNyLhrcFNQEn4xHC8Gz7/jtA4Dnkjtk8AAXJEw8=";
   };
 
-  nativeBuildInputs = [ cmake ];
+  nativeBuildInputs = [
+    cmake
+    catch2
```

Comment:
You need to add catch2 to `buildInputs` too, otherwise the build will fail for `strictDeps = true`. (#178468)

Reply:
can i drop it from nativeBuildInputs then, or does it do build-time checks or something?

Reply:
You can try to remove it, and then test by setting `strictDeps = true;`. If it still builds it is safe to remove.